twoliter/src/cmd/publish.rs

```diff
@@ -0,0 +1,57 @@
+use crate::cargo_make::CargoMake;
+use crate::project;
+use crate::tools::{install_tools, tools_tempdir};
+use anyhow::Result;
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Debug, Parser)]
+pub(crate) enum PublishCommand {
+    Ami(PublishAmi),
+}
+
+impl PublishCommand {
+    pub(crate) async fn run(self) -> Result<()> {
+        match self {
+            PublishCommand::Ami(publish_ami) => publish_ami.run().await,
+        }
+    }
+}
```

Comment:
IMO - we should have the "type" of the publication target be recorded in the variant manifest somehow, so that "AMI" doesn't leak into this interface.

```
# good
twoliter publish
# publishes an AMI for aws-*
# publishes an OVA for vmware-*
# prints a "skipping" or no-op message for unrecognized types

# bad
twoliter publish ami
twoliter publish ova
twoliter publish metal # what would this do?
```

This is starting to get into the question of how we merge `Infra.toml` with whatever shape we would use to specify the VMware equivalent, and pull all that back into `Twoliter.toml`.

I don't think we want to resolve that right now but `twoliter publish ami` feels like a mistake to avoid.

Reply:
> twoliter publish

Reply:
I've removed publishing from this pr.